platform/src/main/java/org/hillview/table/EqualityFilter.java

```diff
@@ -0,0 +1,58 @@
+package org.hillview.table;
+
+import com.sun.tools.javac.util.Assert;
+import org.hillview.table.api.IColumn;
+import org.hillview.table.api.ITable;
+import org.hillview.utils.Converters;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Objects;
+
+/**
+ * This filter maps a given Table to a Table that only contains the given value in the specified column.
+ */
+public class EqualityFilter implements TableFilter {
+    @Nonnull
```

Comment:
My mistake, this is not needed; by default everything is non-null.
Only nullable is needed.